css/styles.css

```diff
@@ -0,0 +1,62 @@
+header {
+  background-color: indigo;
+  font-size: xx-large;
```

Comment:
You should be using `rem` for `font-size`. Review the `effective-css` guide I assigned over the weekend.